### PR TITLE
chore: format code

### PR DESCRIPTION
**Pull request by g-plane**

### CODE REVIEW

**Comment by Aladdin-ADD** [comment]
to do autofix, can we add a precommit script?

File: configs/tsconfig-lint.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [
+    "../packages/*/src"
```
Comment (Aladdin-ADD) [inline configs/tsconfig-lint.json]: need add "../packages/*/test"?

and the filename could better be `tslint-config.json`? 😄

**Comment by KFlash** [comment]
@g-plane Loads of conflicts that you need to resolve

cc / @fkleuver

File: configs/tsconfig-lint.json (hunk 1 above)
Comment (g-plane) [reply configs/tsconfig-lint.json]: Oh, I will add it.
`tslint-config.json` is confusing, and it means the config file of TSLint, not TypeScript itself.

Comment (Aladdin-ADD) [reply configs/tsconfig-lint.json]: sorry, I'm not I got it correctly. the file is for tsc, not for tslint?
so why add a `-lint`?

Comment (g-plane) [reply configs/tsconfig-lint.json]: Yes, it is for tsc. Naming with `lint` can let us distinguish with other configs.

Comment (Aladdin-ADD) [reply configs/tsconfig-lint.json]: emm.. it's very misleading! o((⊙﹏⊙))o.

Comment (Aladdin-ADD) [reply configs/tsconfig-lint.json]: what's the difference? why not just 1 tsconfig?

Comment (g-plane) [reply configs/tsconfig-lint.json]: The compiler options is different between `tsconfig-test` and `tsconfig-build`. Sent from my phone and I didn't check the differences.

**Comment by KFlash** [comment]
This pull request **introduces 1 alert** when merging d5a71975e0398f594e843fc0fb83ab1120f0ab19 into fce154f25d22ca13ba808a7995e5eb727c5aaddd - [view on LGTM.com](https://lgtm.com/projects/g/cherow/cherow/rev/pr-26584ccc78f2c6918c988fe820a9d9c955873af6)

**new alerts:**

* 1 for Useless assignment to local variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by g-plane** [comment]
I'll rebase and re-push this PR later to fix those problems.

File: configs/tsconfig-lint.json (hunk 1 above)
Comment (fkleuver) [reply configs/tsconfig-lint.json]: I don't see the need for a `-lint` for linting. We already have a difference between build and test. Build and lint should have the same config for the same process.

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
 	},
 	"scripts": {
 		"bench": "node --allow-natives-syntax bench/bench.js 100",
+		"lint": "tslint -p ./configs/tsconfig-lint.json",
 		"commit": "git-cz"
```
Comment (fkleuver) [inline package.json]: We already have lint scripts, you can run them from lerna..

File: packages/cherow-cli/src/index.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { parse } from 'cherow';
 import fs from 'fs';
-import program from 'commander';  // tslint:disable-line import-name
+import program from 'commander'; // tslint:disable-line import-name match-default-export-name
 import chalk from 'chalk';
```
Comment (fkleuver) [inline packages/cherow-cli/src/index.ts]: I guess this rule can actually be disabled in tslint.json

**Comment by fkleuver** [comment]
Guys.. I said this before, these autolints are not improvements. We need custom formatting to have the code compact and readable.

Really appreciate the effort, especially since I'm too busy to fix some of these things myself at the moment. But the only formatting issue that ever really needs fixing is redundant lines (e.g. between a jsdoc comment and the function declaration), or inconsistent indentation.

I'm seeing tons of instances where code that was put on a single line by hand, gets expanded out into multiple lines and just makes it harder to read.

Also, when disabling tslint rules, please comment them out instead of removing them. This tslint.json is based on the default that comes with the microsoft recommended ruleset. I very deliberately kept it complete like that so it's easy to revisit the rules, not having to look up the ones that aren't defined, and also to be able to compare ours to the standard and occasionally revisit to try and be more strict with it.

**Comment by KFlash** [comment]
This pull request **introduces 1 alert** when merging a217a00c879ed69cb7e3e2f54694597b0872d0ff into 69d7f689870d2ecc3ee03875981bc4b4e917a97a - [view on LGTM.com](https://lgtm.com/projects/g/cherow/cherow/rev/pr-0ac67da4f166836cda8542d516c0605f133da2b0)

**new alerts:**

* 1 for Useless assignment to local variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request closed without merging**